src/styles/watch.scss

```diff
@@ -0,0 +1,56 @@
+.watch {
+  box-sizing: border-box;
+  width: 500px;
+  height: 500px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border: 8px dotted #fff;
+  border-radius: 100%;
+  transform: rotate(90deg);
```

Comment:
Just because you use `p` tag for watch hands you must rotate your square. It doesn't make sense. You can rotate your watch hands how do you like. So rotate hands